### ISSUE TITLE
go get github.com/kubermatic/kubeone seems broken

### ISSUE THREAD

**Issue by mooperd** [kubermatic/kubeone#742]
Tested on OSX and Ubuntu 18.04 - similar errors.
```
$ go version
go version go1.13.4 darwin/amd64
$ go get github.com/kubermatic/kubeone
package sigs.k8s.io/cluster-api/pkg/apis/cluster/common: cannot find package "sigs.k8s.io/cluster-api/pkg/apis/cluster/common" in any of:
	/usr/local/go/src/sigs.k8s.io/cluster-api/pkg/apis/cluster/common (from $GOROOT)
	/Users/andrew/go/src/sigs.k8s.io/cluster-api/pkg/apis/cluster/common (from $GOPATH)
package sigs.k8s.io/cluster-api/pkg/apis/cluster/v1alpha1: cannot find package "sigs.k8s.io/cluster-api/pkg/apis/cluster/v1alpha1" in any of:
	/usr/local/go/src/sigs.k8s.io/cluster-api/pkg/apis/cluster/v1alpha1 (from $GOROOT)
	/Users/andrew/go/src/sigs.k8s.io/cluster-api/pkg/apis/cluster/v1alpha1 (from $GOPATH)
package sigs.k8s.io/cluster-api/pkg/client/clientset_generated/clientset/scheme: cannot find package "sigs.k8s.io/cluster-api/pkg/client/clientset_generated/clientset/scheme" in any of:
	/usr/local/go/src/sigs.k8s.io/cluster-api/pkg/client/clientset_generated/clientset/scheme (from $GOROOT)
	/Users/andrew/go/src/sigs.k8s.io/cluster-api/pkg/client/clientset_generated/clientset/scheme (from $GOPATH)
(base) Andrews-MacBook-Pro:~ andrew$ go get -u github.com/kubermatic/kubeone
package sigs.k8s.io/cluster-api/pkg/apis/cluster/common: cannot find package "sigs.k8s.io/cluster-api/pkg/apis/cluster/common" in any of:
	/usr/local/go/src/sigs.k8s.io/cluster-api/pkg/apis/cluster/common (from $GOROOT)
	/Users/andrew/go/src/sigs.k8s.io/cluster-api/pkg/apis/cluster/common (from $GOPATH)
package sigs.k8s.io/cluster-api/pkg/apis/cluster/v1alpha1: cannot find package "sigs.k8s.io/cluster-api/pkg/apis/cluster/v1alpha1" in any of:
	/usr/local/go/src/sigs.k8s.io/cluster-api/pkg/apis/cluster/v1alpha1 (from $GOROOT)
	/Users/andrew/go/src/sigs.k8s.io/cluster-api/pkg/apis/cluster/v1alpha1 (from $GOPATH)
package sigs.k8s.io/cluster-api/pkg/client/clientset_generated/clientset/scheme: cannot find package "sigs.k8s.io/cluster-api/pkg/client/clientset_generated/clientset/scheme" in any of:
	/usr/local/go/src/sigs.k8s.io/cluster-api/pkg/client/clientset_generated/clientset/scheme (from $GOROOT)
	/Users/andrew/go/src/sigs.k8s.io/cluster-api/pkg/client/clientset_generated/clientset/scheme (from $GOPATH)
```

**Comment by xmudrii** [comment]
@mooperd KubeOne uses Go modules, so you must have Go modules support enabled. You can enable modules using `export GO111MODULE=on`. After that, you should be able to use `go get` to obtain KubeOne.

**Comment by xmudrii** [comment]
Fixed by #744
/close
